app/decorators/odk/condition_group_decorator.rb

```diff
@@ -7,7 +7,7 @@ def to_odk
       if members.empty?
         negate? ? "false()" : nil
       else
-        conjunction = true_if == "all_met" ? I18n.t("common.and") : I18n.t("common.or")
+        conjunction = true_if == "all_met" ? "and" : "or"
```

Comment:
should this also be case insensitive?

Reply:
oh wait this is not xpath, so it doesn't matter?

Reply:
do you mean the `all_met` part? right that's coming hardcoded from nemo not xpath. but I'll check in on slack before merging in case I'm misunderstanding

Reply:
nm, looks good!